### PR TITLE
Improve get meta

### PR DESCRIPTION
**Pull request by Skwarson96**
* Improve get_meta_file.py - now script search imgs in all directories and add argparse
* Add tensorboard (required while training) and update torch, torchvision in requirements.txt

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -7,8 +7,9 @@ PyYAML==6.0.1
 Requests==2.31.0
 scipy==1.9.1
 timm==0.9.2
-torch==1.7.1
-torchvision==0.8.2
+torch==2.2.0
+torchvision==0.17.0
```
Comment (piddnad) [inline requirements.txt:11]: I'm not sure if pytorch 2.2.0 is compatible with current basicsr, which is essential for training. Have you tested it?
If not, I think it's better to keep these two lines unchanged.
Besides, the [guide](https://github.com/piddnad/DDColor?tab=readme-ov-file#install-with-conda-recommend) in readme file still installs torch 1.7.1.

Comment (Skwarson96) [reply requirements.txt:11]: I tested training and inference (hugging face) and everything was ok

**Comment by Skwarson96** [comment]
Hello, I updated readme

**Comment by piddnad** [comment]
Thanks @Skwarson96 , I will merge this.

**Pull request merged**